app/database/permission_granted_store.go

```diff
@@ -65,3 +69,7 @@ func (s *PermissionGrantedStore) EditIndexByName(name string) int {
 func (s *PermissionGrantedStore) EditNameByIndex(index int) string {
 	return s.PermissionNameByKindAndIndex("edit", index)
 }
+
+func (s *PermissionGrantedStore) permissionsPropagateTableName() string {
+	return golang.IfElse(s.arePropagationsSync(), "permissions_propagate_sync", "permissions_propagate")
```

Comment:
So 2 propagations cannot run in parallel when "sync"? Except for this table name, was it an actual requirement?

Reply:
Nothing stops sync propagations from being run concurrently. The *_sync tables are processed inside transactions, concurrent sessions don’t see rows added by other sessions.